ui/src/containers/Header/index.tsx

```diff
@@ -73,7 +73,17 @@ const Header: React.FC = observer(() => {
   return (
     <React.Fragment>
       <PageHeader
-        logo={<Brand src={logo} alt="Tekton Hub Logo" onClick={homePage} />}
+        logo={
+          <Brand
+            src={
+              CUSTOM_LOGO_BASE64_URI != ''
+                ? `data:image/png;base64,${CUSTOM_LOGO_BASE64_URI}`
```

Comment:
do not force media type to `image/png`, allow custom input.
`CUSTOM_LOGO_BASE64_URI` is not a uri. it is a data.

Reply:
@jkandasa I agree with you, but in that case we might have to restrict with atleast some data types right ?

Reply:
@jkandasa I agree with you, but in that case we might have to restrict with atleast some data types right ?

Reply:
@PuneetPunamiya May be, IMO leaving without a restriction is a fine option. I am not familiar with supported types.